docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -0,0 +1,140 @@
+    </privileges>
+    ```
+
+2.  To use the functions of the Widget Viewer API, include the `<widget_viewer_evas.h>` header file in your application:
+
+    ```
+    #include <widget_viewer_evas.h>
+    ```
+
+<a name="display"></a>
+## Display Widgets
+
+Widget application shares its window to the Widget Viewer application. To display the widget application's buffer you have to create a widget object. The widget application's buffer is displayed on the widget object. You can create the widget object as shown in the following code:
+
+1. Initialize Widget Viewer using `widget_viewer_evas_init()`. The parameter of `widget_viewer_evas_init()` is the window object of an application:
+
+   ```
+   Evas_Object *win = elm_win_add(NULL, "VIEWER", ELM_WIN_BASIC);
+   int ret = widget_viewer_evas_init(win);
+   ```
+
+2. Create a widget object using `widget_viewer_evas_add_widget()`. The parameters of `widget_viewer_evas_add_widget()` are parent Evas_Object, widget ID, content information, and widget update period. The widget ID is the widget application's app ID or class ID. Content information is the bundle encoded data, which contains extra information on the widget. You can retrieve the content information using `widget_viewer_evas_get_content_info()`:
+
+
+   ```
+   const char *content_info;
+   bundle_raw *raw;
+   int len;
+
+   bundle_encode(data, &raw, &len);
+   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
+   evas_object_resize(widget, 100, 100);
+   evas_object_show(widget);
+   ```
+
+3. Finalize Widget Viewer functionality using `widget_viewer_evas_fini()`:
+
+   ```
+   int ret = widget_viewer_evas_fini();
+   ```
+
+<a name="manage_life-cycle"></a>
+## Manage widget life-cycle
+
+After displaying a widget, you can manage the widget application's lifecycle:
+
+1. Create widget object using `widget_viewer_evas_add_widget()`:
+
+   ```
+   Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
+   ```
+
+2. Resume the instance of the widget application using `widget_viewer_evas_resume_widget()` with the widget object as the parameter:
+
+   ```
+   int ret = widget_viewer_evas_resume_widget(widget);
+   ```
+
+3. Pause the instance of the widget application using `widget_viewer_evas_pause_widget()` with the widget object as the parameter:
+
+   ```
+   int ret = widget_viewer_evas_pause_widget(widget);
+   ```
+
+4. Resume or pause all the widget instances displayed by the Widget Viewer application using `widget_viewer_evas_notify_resumed_status_of_viewer()` and `widget_viewer_evas_notify_paused_status_of_viewer()`:
+
+   ```
+   int ret = widget_viewer_evas_notify_resumed_status_of_viewer();
+   ret = widget_viewer_evas_notify_paused_status_of_viewer();
+   ```
+
+<a name="retrieve_information"></a>
+## Retrieve Widget Information
+
+After displaying widget, you can retrieve information about widget instances as shown in the following code:
```

Comment:
How about removing "as shown in the following code:" as the line 75.

Reply:
I agree.
Thank you for pointing this.
From the previous comment https://github.com/Samsung/tizen-docs/pull/986#discussion_r362389792, it was updated in the next commit.
But in the new commits, I think it got missed.

Reply:
@gotoextreme 
I humbly request you to update the comments in the latest file (the file after the last commit).
I see that some of the comments from the previous reviews have been missed in the consecutive commits.

This is a suggestion to avoid missing comments when there are more review comments on a PR.